tools/RAiDER/utilFcns.py

```diff
@@ -281,9 +287,9 @@ def checkShapes(los, lats, lons, hts):
 
     if not test1 and test2:
         raise ValueError(
-         'I need lats, lons, heights, and los to all be the same shape. ' +
-         'lats had shape {}, lons had shape {}, '.format(lats.shape, lons.shape) +
-         'heights had shape {}, and los was not Zenith'.format(hts.shape))
+            'I need lats, lons, heights, and los to all be the same shape. ' +
```

Comment:
BTW you can split string literals across multiple lines like this.
```suggestion
            'I need lats, lons, heights, and los to all be the same shape. '
```

Reply:
I'll make a note to update throughout, as probably there are other places with this that could all be fixed together.